Tiltfile

```diff
@@ -80,12 +80,12 @@ RUN wget --output-document /restart.sh --quiet https://raw.githubusercontent.com
     wget --output-document /start.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh && \
     chmod +x /start.sh && chmod +x /restart.sh
 
-FROM debian:stretch-slim as tilt
+FROM gcr.io/distroless/base:debug as tilt
 WORKDIR /
 COPY --from=tilt-helper /start.sh .
 COPY --from=tilt-helper /restart.sh .
 COPY .tiltbuild/manager .
-ENTRYPOINT [ "/start.sh", "/manager" ]
+ENTRYPOINT [ "sh", "/start.sh "/manager" ]
```

Comment:
We can probably remove this. I believe the way things work with Tilt is that it uses the `entrypoint` from the `docker_build` call, or if that is absent, it uses the `command` and `args` in the pod spec.

It's fine to leave it - it won't hurt anything, but do you want to try removing it?

Reply:
It works if we remove it, that's what I initially did :D